BUCK

```diff
@@ -0,0 +1,22 @@
+genrule(
```

Comment:
@nikhedonia, you should be able to use [export_file](https://buckbuild.com/rule/export_file.html) to export license file and add a dependency on it just like on any other rule.